Output raw data when piped

daviddavis opened the pull request:
[noissue]

mdellweg commented:
Why don't we just switch of the coloring on a tty?
I might want to pipe yaml to a tool that understands it.

daviddavis commented:
The only caveat is that we are also formatting the json but I removed that too by setting indent to `None` if `isatty`. Also, I added an `isatty` propery.

mdellweg commented on tests/scripts/test_output.sh:
```diff
@@ -0,0 +1,14 @@
+}
+trap cleanup EXIT
+
+expect_succ pulp file repository create --name "cli_test_file_repo" --description "Test repository for CLI tests"
```
Is this handled in a way that $OUTPUT looks like it was on a TTY?

daviddavis replied:
Nope, let me try something different.

mdellweg replied:
So you found something?
I'd be happy to take it without a test, because faking a user session is hard...

daviddavis replied:
I did. Check it out.
